_component_design/loading-indicators.md

```diff
@@ -0,0 +1,205 @@
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Loading spinner with text
+
+When text is contained inside a spinner icon, the spinner matches the size of the text. Therefore, spinner size is measured in *em*.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Loading spinner and text used in a primary button. Note that the loading text does not use an ellipses."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-text.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Loading text
+
+In certain instances, such as key-value pairs, or other list style patterns - a text label can be used without a spinner.
+
+{% endcolumn %}
+
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"Loading text shown in a key-value pair list.
+Note that loading text can occur in multiple instances simultaneously."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-text-only.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"Loading text, when used alone, should always use ellipses. By default, the text uses a type-size of 16px."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-text-font.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Animation
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Animation can be used in loading indicators to visually indicate that back-end systems are performing load operations. The following are best practice for animating spinner icons:
+
+- Use a simple animation.
+- Rotate icons 360 degrees in 0.8 seconds.
+- Always rotate icons clockwise.
+
+{% endcolumn %}
+
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"Spinner icons rotate 360 degrees in 0.8sec."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-360.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"Spinner icons must always rotate clockwise."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-rotate-clockwise.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+## Layout
+
+<div class="hxRow">
+
+Loading indicators are generally centered within their given component. The following are examples of loading indicators inside various components of the UI:
```

Comment:
Place this text inside a column to fix alignment and close column prior to open the column on line 165. 

Recommend col-12 or col-8